data/sql/updates/pending_db_characters/rev_1707981003566465200.sql

```diff
@@ -0,0 +1,3 @@
+--
+ALTER TABLE `recovery_item`
+    ADD COLUMN `DeleteDate` INT(10) UNSIGNED NULL DEFAULT NULL AFTER `Count`;
```

Comment:
it's good to add a newline at the end of the file, moreover `INT(10)` is not necessary anymore, you can put just `INT` ( [source](https://stackoverflow.com/questions/58938358/mysql-warning-1681-integer-display-width-is-deprecated) ).